package-lock.json

```diff
@@ -1,11194 +1,8 @@
 {
   "name": "@triniti/cms",
-  "version": "0.13.5",
-  "lockfileVersion": 2,
+  "version": "0.13.6",
+  "lockfileVersion": 1,
```

Comment:
@joshlevy update npm so the lockFileVersion doesnt revert https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion